_episodes/03-create.md

```diff
@@ -644,7 +643,7 @@ Given that there is no way to retrieve files deleted using the shell,
 
 Oftentimes one needs to copy or move several files at once.
 This can be done by providing a list of individual filenames,
-or specifying a naming pattern using wildcards.
+or specifying a naming pattern using wildcards. Wildcards are special characters that can be used to represent unknown characters or sets of characters when navigating the Unix file system.
```

Comment:
Please add line breaks into this long line.

Reply:
OK, I will fix it.